src/script/page/LeftSidebar/panels/StartUI/PeopleTab.tsx

```diff
@@ -27,7 +27,7 @@ import {partition} from 'underscore';
 import {WebAppEvents} from '@wireapp/webapp-events';
 
 import {Icon} from 'Components/Icon';
-import {UserList, UserlistMode} from 'Components/UserList';
+import {UserList, UserlistMode} from 'Components/UserList/UserList';
```

Comment:
```suggestion
import {UserList, UserlistMode} from 'Components/UserList';
```

Isn't there a `index.js` file in there?